New version 0.25.1 breaks JupyterLab CI

https://github.com/jupyterlab/jupyterlab/actions/runs/8538604184/job/23391600562?pr=16078

#### `Install from SDist and Test`:

```
+ uv pip compile pyproject.toml -o /home/runner/constraints.txt
error: failed to read from file `pyproject.toml`
  Caused by: No such file or directory (os error 2)
```

#### `Test Minimum Versions`:

```
Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' for action 'jupyterlab/maintainer-tools/.github/actions/install-minimums@v1'.
```

#### `Windows (integrity)` and `Windows (python)`:

```
Run .\scripts\ci_install

    Directory: C:\Users\runneradmin

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d----            4/3/[20](https://github.com/jupyterlab/jupyterlab/actions/runs/8538604175/job/23391595403?pr=16078#step:4:21)24 12:24 PM                .jupyter
ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/c/Users/runneradmin/constraints.txt'
Exception: D:\a\jupyterlab\jupyterlab\scripts\ci_install.ps1:12
Line |
  12 |  … TCODE -ne 0) { throw "Command failed. See above errors for details" }
     |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Command failed. See above errors for details
Error: Process completed with exit code 1.
```

--- a/.github/actions/base-setup/action.yml
+++ b/.github/actions/base-setup/action.yml
@@ -132,8 +132,12 @@ runs:
         elif [ $DEPENDENCY_TYPE == 'minimum' ]; then
           FLAGS="--resolution=lowest-direct"
         fi
-        uv pip compile $FLAGS pyproject.toml -o $HOME/constraints.txt
-        echo "PIP_CONSTRAINT=$HOME/constraints.txt" >> $GITHUB_ENV
+        if [ -f pyproject.toml ]; then
+          uv pip compile $FLAGS pyproject.toml -o $HOME/constraints.txt
+          echo "PIP_CONSTRAINT=$HOME/constraints.txt" >> $GITHUB_ENV
+        elif [ -n "$FLAGS" ]; then
+          echo "Missing pyproject.toml needed for DEPENDENCY_TYPE: $DEPENDENCY_TYPE"
+        fi
 
     - name: Print Diagnostics
       shell: bash

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-    "name": "@jupyterlab/maintainer-tools",
-    "version": "0.25.1",
-    "private": true
+  "name": "@jupyterlab/maintainer-tools",
+  "version": "0.25.1",
+  "private": true
 }

